Proposals/TypeNetwork_ParametricAxes/ProposalSummary_ytas.md

```diff
@@ -38,7 +38,7 @@ in coordination leading and column width to prevent letters from touching vertic
 **UI recommendations:** Users may choose to program a variant in connection to direct or
 conjunctive input for a page description language, or via a user interface.
 
-**Script or language considerations:** Can be used for all scripts.
+**Script or language considerations:** This axis is primarily intended for Latin script and other scripts that may have ascender elements intended to remain aligned relative to the Latin ascenders throughout the variable space. Typically, this would apply to scripts in which some letters generally have a smaller core height than the ascender that is being aligned with the Latin ascender.
```

Comment:
This wording is unclear:

"generally have a smaller core height than the ascender that is being aligned with the Latin ascender"

The core height is the thing that is aligning with the Latin ascender? That sounds unlikely.

Can you provide clearer wording?